src/utils/AccessibleButton.ts

```diff
@@ -52,8 +52,8 @@ export class AccessibleButton {
     return this;
   }
 
-  public withoutLabel() {
-    this.hasLabel = false;
+  public withoutLabelOrTitle() {
```

Comment:
In the same theme as Benjamin's comment above, I think a better name for this function based on the logic would be `prioritizeTitleOverLabel`.

However, this raises another question. Wouldn't it be simpler to just not call `.withLabel` when building an accessible button if one is also calling `.withTitle`?

It seems like the current approach is fixing an issue of two method calls by adding a third method call.

Reply:
Hmm, this method removes the required label/title rule, and prevents an console error (because usually a button without either is not accessible enough).

> However, this raises another question. Wouldn't it be simpler to just not call .withLabel when building an accessible button if one is already calling .withTitle?

You could, but putting both causes a repetition and isn't really accessible either. Those are mostly tools for us the devs.